wallet/src/de/schildbach/wallet/database/entity/BlockchainIdentityData.kt

```diff
@@ -64,7 +65,13 @@ data class BlockchainIdentityData(var creationState: CreationState = CreationSta
         }
         if (wallet != null) {
             creditFundingTransactionCache = wallet.getTransaction(creditFundingTxId)?.run {
-                wallet.getCreditFundingTransaction(this)
+                // TODO: is this the right way to get the auth extension?
+                // Should we have static ID for the extension?
+                // Do we need to call addKeyChains here?
+                val authExtension = wallet.addOrGetExistingExtension(
+                    AuthenticationGroupExtension(wallet.params)
+                ) as AuthenticationGroupExtension
+                authExtension.getCreditFundingTransaction(this)
```

Comment:
Sadly, this is the way.  having a static id will be the best way for this case.  I will add it in my Platform 0.24 story.  When CoinJoin (`feature-coinjoin`) is merged into DashPay (`dashpay`), then we replace this and other places.

https://github.com/dashpay/dashj/pull/216